File: leetcode/412. Fizz Buzz/soln.cpp
class Solution {
public:
    vector<string> fizzBuzz(int n) {
        vector<string> soln{};
        std::string  f{"Fizz"};
        std::string b{"Buzz"};
        std::string fb{f+b};
        for(auto i{1}; i<=n; i++){
            if(i%3 == 0 && i%5 == 0){
                soln.push_back(fb);
            } else if(i% 3 == 0){
                soln.push_back(f);
            } else if(i% 5 == 0){
                soln.push_back(b);
            } else {
                soln.push_back(std::to_string(i));
            }
        }
        return soln;
    }
};

File: leetcode/problems/412. Fizz Buzz/soln.py
class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        soln=[]
        for i in range(1,n+1):
            if(i%3==0 and i%5==0):
                soln.append("FizzBuzz")
            elif(i%3==0):
                soln.append("Fizz")
            elif(i%5==0):
                soln.append("Buzz")
            else:
                soln.append(str(i))
        return soln
        